README.md

```diff
@@ -31,6 +31,7 @@ navigating to the `corrode` directory, installing the `happy` and `alex` tools,
 and then building and installing the `corrode` package:
 
 ```
+cabal update
```

Comment:
I see this present in [other PR](https://github.com/jameysharp/corrode/pull/147/files#diff-04c6e90faac2675aa89e2176d2eec7d8R34) too.